### ISSUE TITLE
Gitlab Input: Can't use IRC Channel names with a dot in the name

### ISSUE THREAD

**Issue by fleaz** [fleaz/CptHook#30]
```
* 'groups[mag][0]' expected type 'string', got unconvertible type 'map[string]interface {}'
FATA[0000] Failed to unmarshal channelmapping into struct
``` 

Config:
```
gitlab:
        enabled: true
        default: "#mag.lab"
        groups:
            "mag.lab":
                - "#mag.lab"
            "fffd":
                - "#fffd"
```

The problem appears to be the "mag.lab" in line 5. Without the dot everything works as expected

**Comment by fleaz** [comment]
Appears to be a issue in viper: https://github.com/spf13/viper/issues/324

**Comment by fleaz** [comment]
Fixed in 4dd163e45fc17643138abc19465657ff6d06283a
